src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md

```diff
@@ -0,0 +1,166 @@
+in many cases, already familiar with the brand or service and have a high level of intentionality
+that results in higher than average engagement metrics.
+
+The Play Store is a store for Android apps, and developers often want to open their Progressive Web
+Apps from their Android apps.
+
+Trusted Web Activity is an open standard that allows browsers to provide a fully web platform
+compatible container that developers can use to render their PWA inside an Android app. The feature
+is available in [Chrome][2], in development in [Firefox Preview Nightly][3], with other browsers
+joining soon.
+
+### Existing solutions were limited
+
+It has always been possible to include web experiences in an Android app, using technologies like
+the [Android WebView][4] or frameworks like [Cordova][5].
+
+The limitation with Android WebView is that it's not intended as a browser replacement. The Android
+WebView is a developer tool for using web UI in an Android app and it doesn't provide complete
+access to modern web platform features such as [contact picker][6], [filesystem][7], among others.
+
+Cordova was designed to augment the shortcomings of WebView, but the APIs are then limited to the
+Cordova environment.  That means you need to maintain an additional codebase for using Cordova APIs
+for your Android app, separate from your PWA on the open web.   
+
+In addition, feature discoverability often doesn't always work as expected and compatibility issues
+between Android versions and OEMs can also be a problem. When using one of those solutions,
+developers need additional quality assurance processes and incur in extra development cost to
+detect and create workarounds.
+
+### Trusted Web Activity is a new container for Web apps on Android
+
+Developers can now use a [Trusted Web Activity][8] as a container to include a PWA as a launch
+activity for an Android app. The technology leverages the browser to render the PWA in full screen,
+ensuring the Trusted Web Activity has the same compatibility with the Web Platform features and
+APIs that the underlying browser does.  There are also open source utilities to make implementing
+an Android app using a Trusted Web Activity even easier.  
+
+Another advantage not available in other solutions is that the container shares storage with the
+browser. Login states and users preferences are shared seamlessly across experiences.
+
+#### Browser Compatibility
+
+The feature has been available in Chrome, since version 75, with Firefox implementing it in their
+nightly version. Other browsers are expected to follow soon.
+
+### Quality Criteria
+
+Web developers should use a Trusted Web Activity when they want to include web content in an
+Android app.  Web content in a Trusted Web Activity must meet PWA install quality criteria and
+additional Android-specific criteria such as [policy compliance][9].
+
+<figure class="w-figure w-figure--center">
+  <img src="./lh-score.png" style="max-width: 100%;"
```

Comment:
```suggestion
  <img src="lh-score.png" style="max-width: 100%;"
```

Reply:
The link was broken in the [current preview](https://deploy-preview-2388--web-dev-staging.netlify.com/using-a-pwa-in-your-android-app/). This is how its done in other articles.